chapters/interface.tex

```diff
@@ -532,6 +532,8 @@ \section{Function-Compatibility or Function-Subtyping for Functions}\label{funct
 \item
   A public input component of \lstinline!A! not present in \lstinline!B! must have a binding
   assignment.
+\item
+  If \lstinline!A! is impure, then \lstinline!B! must also be impure, \cref{pure-modelica-functions}.
```

Comment:
Seems like the bare `\cref` should either be parenthesized or be better embedded:
```suggestion
  If \lstinline!A! is impure, then \lstinline!B! must also be impure, compare \cref{pure-modelica-functions}.
```